string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,10 @@
+/* exported getWords */
+function getWords(string) {
+  var newString = string.split(' ');
+  for (var i = 0; i < newString.length; i++) {
+    if (string.length === 0) {
+      return [];
+    }
```

Comment:
You should do this first in your function, this is a "guard" that checks if the input is empty, and if so, just gives back an empty array.